src/PKSim.UI/Binders/ChartDataBinder.cs

```diff
@@ -76,6 +72,10 @@ public void Bind(ChartData<TX, TY> chartData, PopulationAnalysisChart analysisCh
          {
             var dummySeries = new Series("dummy", _viewType);
             _view.Chart.Series.Add(dummySeries);
+
+            //Zoom initialization should be performed with an existing Diagram
+            if (_allowZoom)
```

Comment:
Moved this line to place with existing diagram. This initialized everything needed